improver/utilities/call_object_method.py

```diff
@@ -0,0 +1,31 @@
+# (C) Crown Copyright, Met Office. All rights reserved.
+#
+# This file is part of 'IMPROVER' and is released under the BSD 3-Clause license.
+# See LICENSE in the root of the repository for full licensing details.
+
+"""module to give access to callable methods on objects."""
+
+
+def call_object_method(obj: object, method_name: str, **kwargs):
+    """
+    Calls a method on an object with the supplied arguments.
+
+    This method allows us to construct a callable method for DAGRunner to execute
```

Comment:
I wouldn't reference what library you choose to execute these with.
```suggestion
    This method allows us to construct a callable method for the caller to execute
```